src/ci/resources/enable_metadata_archiving.inc.conf

```diff
@@ -21,6 +21,9 @@ services.MetadataService {
       # How long to require after workflow completion before going ahead with archiving:
       archive-delay = 10 minutes
 
+      # How often to send instrumentation metrics (eg. workflows left to archive metric)
+      instrumentation-interval = 1 minute
```

Comment:
TOL this is kind of a generic name for a very specific metric (eg none of the other metrics are affected by this)

Reply:
I was on the fence about keeping this generic v/s specific. I went with generic in case we might want to use this somewhere else in archiving as well for some other metric. If you have strong preference to keep it specific then I can change it